### PR TITLE
Mhryb/tdi 49704 fix tdit4866 http client die on error true status 500

### PR DESCRIPTION
**Pull request by mhryb**

### CODE REVIEW

File: tuj/java/Internet/TDIT4866_HttpClient_DieOnError_true_Status_500/process/TDIT4866_HttpClient_DieOnError_true_Status_500_0.1.item [hunk 1]
```diff
@@ -554,18 +444,130 @@
       <column comment="" key="false" length="-1" name="body" nullable="true" originalLength="-1" pattern="" precision="-1" sourceType="" type="id_String" usefulColumn="true"/>
     </metadata>
   </node>
-  <node componentName="tAssert" componentVersion="0.101" offsetLabelX="0" offsetLabelY="0" posX="288" posY="544">
-    <elementParameter field="TEXT" name="UNIQUE_NAME" show="false" value="tAssert_1"/>
-    <elementParameter field="TEXT" name="DESCRIPTION" value="&quot;Check that HTTPClient is successfull - Status 500 - Die on Error true&quot;"/>
-    <elementParameter field="MEMO_JAVA" name="EXPRESSION" value="((String)globalMap.get(&quot;tHTTPClient_1_ERROR_MESSAGE&quot;))!=null &amp;&amp; ((String)globalMap.get(&quot;tHTTPClient_1_ERROR_MESSAGE&quot;)).contains(&quot;Response status code was not 2xx&quot;)"/>
+  <node componentName="tMysqlOutput" componentVersion="0.102" offsetLabelX="0" offsetLabelY="0" posX="1152" posY="256">
+    <elementParameter field="TEXT" name="UNIQUE_NAME" show="false" value="tMysqlOutput_1"/>
+    <elementParameter field="CHECK" name="ACTIVATE" value="false"/>
```
Comment (dmytro-sylaiev) [inline tuj/java/Internet/TDIT4866_HttpClient_DieOnError_true_Status_500/process/TDIT4866_HttpClient_DieOnError_true_Status_500_0.1.item:449]: see a pb here, please activate this component back

**Review by dmytro-sylaiev — CHANGES_REQUESTED** [review]
Please activate back the tMysqlOutput

File: tuj/java/Internet/TDIT4866_HttpClient_DieOnError_true_Status_500/process/TDIT4866_HttpClient_DieOnError_true_Status_500_0.1.item (hunk 1 above)
Comment (mhryb) [reply tuj/java/Internet/TDIT4866_HttpClient_DieOnError_true_Status_500/process/TDIT4866_HttpClient_DieOnError_true_Status_500_0.1.item]: Thank you for the notice! I fixed

**Review by dmytro-sylaiev — APPROVED** [review]
LGTM

**Pull request merged**